tests/console/yast2_i.pm

```diff
@@ -18,13 +18,34 @@ use testapi;
 use utils;
 use version_utils qw(is_tumbleweed is_jeos is_sle);
 
+sub set_action {
+    my $action = shift;
+    return unless defined($action);
+    assert_script_run qq{sed -i 's/PKGMGR_ACTION_AT_EXIT=.*/PKGMGR_ACTION_AT_EXIT="$action"/' /etc/sysconfig/yast2};
+}
+
 sub run {
     my $self        = shift;
     my $pkgname     = get_var("PACKAGETOINSTALL_RECOMMENDED", "yast2-nfs-client");
     my $recommended = get_var("PACKAGETOINSTALL_RECOMMENDED", "nfs-client");
+    my $output;
+    my $action;
 
     select_console 'root-console';
 
+    if (!script_run 'test -f /etc/sysconfig/yast2') {
```

Comment:
```suggestion
    if (assert_script_run 'test -f /etc/sysconfig/yast2') {
```
your approach will work too , but just wanted to mention another one which will allow to drop `else` below

Reply:
The file is missing in jeos, therefore I will rework this part. However I wanted to ease reviewers life, so die with an explicit message is better.